pkgs/tools/security/theharvester/default.nix

```diff
@@ -47,6 +47,7 @@ python3.pkgs.buildPythonApplication rec {
     shodan
     slowapi
     starlette
+    ujson
```

Comment:
`ujson` is already present (line 40).

Reply:
I don't understand why it is not found when I launch the application... Can you test it in a fresh environment? `nix-shell -p theharvester` and `theHarvester`